Jenkinsfile

```diff
@@ -51,6 +51,7 @@ node('rhel7'){
         dir ('vscode-quarkus') {
             def packageJson = readJSON file: 'package.json'
             sh "vsce package -o ../vscode-quarkus-${packageJson.version}-${env.BUILD_NUMBER}.vsix"
+            sh "npm pack && mv vscode-quarkus-${packageJson.version}.tgz vscode-quarkus-${packageJson.version}-${env.BUILD_NUMBER}.tgz"
```

Comment:
wondering why there is a ../ in previous line. Is it changing something here?

Reply:
this step is running inside the vscode-quarkus directory. It's putting the vsix file in the parent dir, so you'll need to do the same

Reply:
Thanks for the explanation. I modified, to also move the generated tgz source files to the parent dir.